src/chorizo-and-chickpea-soup.md

```diff
@@ -0,0 +1,30 @@
+# Chorizo & Chickpea Soup
+
+Recommend serving with corn bread.
+
+- 🍳 Cook time: 10-15 min
+
+## Ingredients
+
+- Chorizo Sausage(s)
+- 2 Garlic cloves
+- Ginger
+- Paprika, 2 Tsp
+- Cumin, Parsley
+- Juice from whole Lemon
+- Chickpeas, 400g ( DRY )
+- Butter and sunflower oil, for frying
+
+## Directions
+
+1. Fry the sliced Chorizo in butter, alongwith diced Ginger and Garlic. Add oil after a couple mins.
+2. Pour in lemon-juice, chickpeas, and Cumin & Paprika. Then 100ml of water.
+3. Allow to boil for 5 or so min. Add Parsley and then salt to taste.
```

Comment:
Still need to fix these issues I mentioned in the previous PR

Reply:
Sorry, what are the issues?

Reply:
If you look in the previous PR, I talked about capitalization inconsistencies + spelling errors. Thanks

Reply:
This file is quite different from the original as I modified it a fair bit. Could you lemme know where the spelling errors and such are, perhaps by making proposed edits?

Reply:
https://github.com/LukeSmithxyz/based.cooking/pull/605#discussion_r622219487, https://github.com/LukeSmithxyz/based.cooking/pull/605#discussion_r622220053, https://github.com/LukeSmithxyz/based.cooking/pull/605#discussion_r622220566

Reply:
Yes that's unclear since the file has been changed and you did not specify much to begin with. I'm gonna assume you want me to capitalise as nouns 'lemon-juice' and 'chickpeas', in addition to changing 'alongwith' to "along with". Is there anything else?

Reply:
> Yes that's unclear since the file has been changed and you did not specify much to begin with. I'm gonna assume you want me to capitalise as nouns 'lemon-juice' and 'chickpeas', in addition to changing 'alongwith' to "along with". Is there anything else?

That's fine. I'm saying that you had capitalized words at the end of sentences that didn't really need to be capitalized. Thanks.

Reply:
Oh did you mean like ( DRY )? I'll go through it and sort it out.